### PR TITLE
Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by emmamarichal**
008eb82: [gftools-packager] Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

* Micro 5 Charted Version 1.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/scfried/soft-type-micro at commit https://github.com/scfried/soft-type-micro/commit/b642aae8b850ae551a93623e3c116866de402879.

 ba36965: Description and metadata

### CODE REVIEW

**Comment by emmamarichal** [comment]
🔥 FAIL: Check font follows the Google Fonts vertical metric schema

🔥 FAIL: Checking OS/2 usWinAscent & usWinDescent.
-> Need to be aligned to the charted version (we can't overlap the grid)

🔥 FAIL: Whitespace glyphs have ink?
-> There is a grid in background, so no whitespace

⚠ WARN: METADATA.pb: Designers are listed correctly on the Google Fonts catalog?
-> Working on it

⚠ WARN: Does GPOS table have kerning information? This check skips monospaced fonts as defined by post.isFixedPitch value
-> No kerning, to be aligned to the charted version (we can't overlap the grid)

**Comment by vv-monsalve** [comment]
@emmamarichal, inspecting at this font, it seems that ascenders and descender values are indeed using (or based on) pixel-grid units. So the question would be if the designer could change the proportion. E.g., Ascenders to go from 2 to 3x and descenders from 1 to 2, or any combination of grid-pixel equivalent values that could prevent collisions affecting legibility.

Screen Shot 2023-12-12 at 16 40 09

Screen Shot 2023-12-12 at 16 46 35

**Comment by emmamarichal** [comment]
I agree, I'll ask to the designer if she would be ok to do that modifications! :)

**Comment by emmamarichal** [comment]
Updated

## Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

---
 457e65c: [gftools-packager] Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

* Micro 5 Charted Version 1.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/scfried/soft-type-micro at commit https://github.com/scfried/soft-type-micro/commit/032a7f1329bd7c2bb30aa727a8c48e84c780f41d.

 263198b: description and metadatas

**Comment by emmamarichal** [comment]
@vv-monsalve updated!

File: ofl/micro5charted/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<p>Soft Type is a collection of typefaces designed for knitting color-work. Each typeface has a “Regular” and “Charted” version and some include multiple scales so you can fit type on your knits, no matter the project's size.</p>
+
+<p>Designing typefaces for knitting is essentially the same as designing pixel type. However, in practice, the pixel size is determined by the properties of the yarn in use. Scale is determined by the weight of the yarn as well as the number of pixels that make up the height of each letter. In this collection, the number following the name of the typeface indicates the height of the capital letters for that typeface.</p>
+
+<p>These typefaces were designed with machine knitting in mind, but could be used for hand knitting, needlework, bedazzling, or many other textile crafts.</p>
```
Comment (vv-monsalve) [inline ofl/micro5charted/DESCRIPTION.en_us.html:5]: Same comment as in Jacquard 12

File: ofl/micro5charted/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Soft Type Project Authors (https://github.com/scfried/soft-type-micro)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```
Comment (vv-monsalve) [inline ofl/micro5charted/OFL.txt:5]: Url for OFL must be updated to https://openfontlicense.org/

**Comment by vv-monsalve** [comment]
@emmamarichal vertical metrics look better now. However, as part of our requirements, the capital letters are expected to be centered in the line since this is commonly expected in web design.
Given that the vertical metrics values are not allowed to be changed once the font is published, perhaps you and @scfried would like to review and confirm if you are adding another "module" (pixel grid) in the descenders.
  
Screen Shot 2024-01-17 at 18 31 08

**Comment by emmamarichal** [comment]
Updated

## Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

---
 0c703b1: [gftools-packager] Micro 5 Charted: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

* Micro 5 Charted Version 1.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/scfried/soft-type-micro at commit https://github.com/scfried/soft-type-micro/commit/e8450033cd07542081f72abd540b93574434bc71.

 fe7ae84: description and metadatas updated

**Comment by emmamarichal** [comment]
@vv-monsalve 
both versions updated with
- v-metrics (caps centered)
- url in licence

**Comment by vv-monsalve** [comment]
> 🔥 FAIL Glyph 'uni00A0' has ink. It needs to be replaced by an empty glyph. [code: has-ink]

This is a non-visible character to prevent automatic like break ([wiki](https://en.wikipedia.org/wiki/Non-breaking_space)).
This one should be empty.

**Comment by emmamarichal** [comment]
@vv-monsalve 
For the non-breaking space, is the character we use for example in this kind of case I think:
`In french, there is a space before a colon : and we use a non breaking space here.`

So, in that case, it needs to have a grid right?

**Comment by vv-monsalve** [comment]
After a pair review, we confirmed that `nbspace` should be identical to the `space` character. That is, in cases in which the space has outlines, the `nbspace` should also have them.

**Comment by vv-monsalve** [comment]
@emmamarichal this font still needs to be included in the categorization tags list.

**Comment by emmamarichal** [comment]
@vv-monsalve I filled the form, it should be good :)

**Review by vv-monsalve — APPROVED** [review]

**Pull request merged**